crates/api/Cargo.toml

```diff
@@ -15,3 +15,6 @@ base64 = { workspace = true }
 bytes = { workspace = true }
 serde = { workspace = true }
 serde_json = { workspace = true }
+
+[dev-dependencies]
+tokio = { version = "1.41.1", features = ["full"] }
```

Comment:
I wonder what y'all think about this.

In order to reign in dependencies, I have been adding justification documentation to the workspace Cargo.toml for each dep I add there, and then was planning on including any dev-dependencies manually in each crate that needs them without said justification.

But if this is too much dev friction, I understand that too, and we can switch to having dev dependencies in the workspace as well.

Reply:
That makes sense to me. How's the wording here: https://github.com/holochain/kitsune2/pull/5/commits/d05c4705b681a955657d9abe56a27b2475aeb642?